lib/geoengineer/gps/gps.rb

```diff
@@ -192,16 +110,20 @@ def nodes
     @_nodes
   end
 
+  def finder
+    @finder ||= GeoEngineer::GPS::Finder.new(nodes)
```

Comment:
It might be cool to initialize the global finder with the current `GEO_ENV`, so that you could use `project::config:type:name`

Reply:
I think that would be for the `node_context` to do with its finder.

Reply:
nah, cuz in non GPS files there is no node context.

Reply:
I meant the `GeoEngineer::GPS::NodesContext` for partials. But for outside those I think adding environment variables makes things complicated and would like to leave out the core of GPS. the environment could create a finder though, with its own context.